source/_patterns/01-molecules/23-hotel-category-images/hotel-category-images.mustache

```diff
@@ -1,7 +1,7 @@
-<div class="mot_stars">
+<sup class="mot_stars">
   {{>atoms-hotel-category-image_filled}}
   {{>atoms-hotel-category-image_filled}}
   {{>atoms-hotel-category-image_filled}}
   {{>atoms-hotel-category-image_half-empty}}
   {{>atoms-hotel-category-image_empty}}
-</div>
+</sup>
```

Comment:
Will the stars be shown as superscript on all different places they appear? Or only when they are added within a text body?

Reply:
Yes. this is already approved by Ankush. Let's let him do further cross-browser testing after the code review. :)

Reply:
If we are sure about that, I can approve this PR.